sos/report/plugins/origin.py

```diff
@@ -40,14 +40,7 @@ class OpenShiftOrigin(Plugin):
     files = None  # file lists assigned after path setup below
     profiles = ('openshift',)
 
-    option_list = [
-        PluginOpt('diag', default=True,
-                  desc='Collect oc adm diagnostics output'),
-        PluginOpt('diag-prevent', default=True,
-                  desc='Use --prevent-modification with oc adm diagnostics'),
-        PluginOpt('all-namespaces', default=False,
-                  desc='collect dc output for all namespaces')
-    ]
+    option_list = []
```

Comment:
No need of this line, as `Plugin` class already sets that (https://github.com/sosreport/sos/blob/main/sos/report/plugins/__init__.py#L551).

Reply:
Ok, I will remove that line. Thx